src/sandstorm/run-bundle.c++

```diff
@@ -1418,7 +1478,7 @@ private:
     // Create the mongo user if it hasn't been created already.
     maybeCreateMongoUser(config);
 
-    context.warning("** Back-end and Mongo started; now starting front-end...");
+    context.warning("** Back-end zand Mongo started; now starting front-end...");
```

Comment:
Er, I introduced and => zand to make sure I was running my modified binary. So ignore this changed line.